src/core/Song.cpp

```diff
@@ -235,9 +235,6 @@ void Song::processNextBuffer()
 			return;
 	}
 
-	// If we have no tracks to play, there is nothing to do
-	if (trackList.empty()) { return; }
```

Comment:
Just curious, should we actually process `Song` if there's nothing? Wouldn't it be better to set a flag on export on empty?

Reply:
We need to export the song in order to increase the play makers. If we never increase them, we can never get the signal that the song is finished.